docs/components/sensor/ultrasonic.md

```diff
@@ -4,13 +4,18 @@ linkTitle: "ultrasonic"
 weight: 60
 type: "docs"
 description: "Configure an ultrasonic model sensor."
-tags: ["sensor", "components"]
+tags: ["sensor", "components", "ultrasonic"]
 icon: "/icons/components/sensor.svg"
 images: ["/icons/components/sensor.svg"]
 # SME: #team-bucket
 ---
 
-Configure an `ultrasonic` sensor to integrate an [HC-S204 ultrasonic distance sensor](https://www.sparkfun.com/products/15569) into your robot:
+{{< alert title="Tip" color="tip" >}}
+An ultrasonic distance sensor can also be configured as a [camera](/components/camera/) resource.
+[Configure it as such](/components/camera/ultrasonic/) to utilize the [camera API](/components/camera/#api) with your ultrasonic sensor hardware, enabling you to use it as an input to a [vision service](/services/vision/) model that returns obstacles.
```

Comment:
I think this might be slightly better
```suggestion
An ultrasonic distance sensor can also be configured as a [camera](/components/camera/) resource.
When configured as a camera, you can use the [camera API](/components/camera/#api) and you can use the component as an input to a [vision service](/services/vision/) model that returns obstacles.
```

Reply:
Done!